### ISSUE TITLE
Setting initial zoom of forceNetwork

### ISSUE THREAD

**Issue by pemorris** [christophergandrud/networkD3#125]
Thanks for a truly great package, 

I was wondering if there was any way of setting the initial zoom of a networkD3 forceNetwork plot? Zooming via the mouse works perfectly, but it would really useful to set a start zoom at the function call. 

Regards

**Comment by MikeBadescu** [comment]
Agree. `linkDistance` can be used to increase separation between points, but it is not sufficient as the text size will not change.

**Comment by higor-gomes93** [comment]
Any updates on this? Maybe there is a way of doing it with the customJS function?
